primitives/src/policy.rs

```diff
@@ -58,6 +58,22 @@ impl Policy {
     /// The maximum allowed size, in bytes, for a micro block body.
     pub const MAX_SIZE_MICRO_BODY: usize = 100_000;
 
+    // Maximum lengths for transaction sender_data and recipient_data fields used during deserialization.
+    /// Maximum size for the sender data field of the transactions.
+    /// The only transactions using this field are RemoveStake and Delete Validator.
+    pub const MAX_TX_SENDER_DATA_SIZE: usize = 1;
```

Comment:
Added tests to both. Thank you, caught another miscalculation, the max recipient data was too small because the Merkle proof size was not being well accounted for.